prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+CP1404/CP5632 Practical
+Counting the different words in a user string input
+"""
+
+word_to_count = {}
+
+user_string = input("Input String: ")
+words = user_string.split()
+for word in words:
+    if word in word_to_count:
```

Comment:
You could also have done this a little easier by using the .get method from dictionaries. Just create a variable called frequence = word_to_count.get(word, 0) in a for loop then call the dict words_to_count[word] = frequency + 1